backend/btrixcloud/crawlconfigs.py

```diff
@@ -489,27 +578,48 @@ async def get_crawl_configs(
                         }
                     }
                 },
+                {"$unset": ["firstSeedFormatted"]},
```

Comment:
Is 'firstSeedFormatted' being used still?

Reply:
nope! that's a vestige from an earlier approach. removed now